draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/TitleBlockZen.spec.tsx

```diff
@@ -642,6 +643,52 @@ describe("<TitleBlockZen />", () => {
     })
   })
 
+  describe("survey status", () => {
+    it("it doesn't render tag when field is omitted", async () => {
+      render(<TitleBlockZen title="Test Title">Example</TitleBlockZen>)
+
+      await waitFor(() => {
+        expect(
+          screen.queryByTestId("survey-status-tag")
+        ).not.toBeInTheDocument()
+      })
+    })
+
+    const surveyStatuses = ["draft", "live", "closed", "scheduled"]
+    const statusClassNameMap = {
+      draft: "statusDraft",
+      live: "statusLive",
+      closed: "statusClosed",
+      scheduled: "statusClosed",
+    }
+    surveyStatuses.forEach(status => {
```

Comment:
Nice work testing these statuses!

I think it's preferable to use the native `it.each` functionality here, [see documentation](https://jestjs.io/docs/api#testeachtablename-fn-timeout).